tests/e2e/specs/setup-mc/step-4-complete-campaign.test.js

```diff
@@ -219,78 +203,6 @@ test.describe( 'Complete your campaign', () => {
 				await completeCampaign.goto();
 			} );
 
-			test.describe( 'Select audience', () => {
-				test( 'should see only three country tags in country input search box', async () => {
-					const countrySearchBoxContainer =
-						getCountryInputSearchBoxContainer( page );
-					const countryTags =
-						getCountryTagsFromInputSearchBoxContainer( page );
-					await expect( countryTags ).toHaveCount( 3 );
-					await expect( countrySearchBoxContainer ).toContainText(
-						'United States'
-					);
-					await expect( countrySearchBoxContainer ).toContainText(
-						'Taiwan'
-					);
-					await expect( countrySearchBoxContainer ).toContainText(
-						'United Kingdom'
-					);
-				} );
-
-				test( 'should only allow searching for the same set of the countries selected in step 2, which is returned by target audience API', async () => {
-					const treeSelectMenu = getTreeSelectMenu( page );
```

Comment:
This util function doesn't seem to be used elsewhere and can be removed.

Reply:
Removed https://github.com/woocommerce/google-listings-and-ads/pull/2551/commits/50b13dfe6c753b74862959a500ab881f1f33285d#diff-ea7e403da1892c5b38977f0659716dca9e04171d9a17f4eb2e2d15dc4d704f85L90